subprojects/groovy-json/src/main/java/groovy/json/JsonOutput.java

```diff
@@ -61,7 +61,7 @@
     private static final String NULL_VALUE = "null";
     private static final String JSON_DATE_FORMAT = "yyyy-MM-dd'T'HH:mm:ssZ";
     private static final String DEFAULT_TIMEZONE = "GMT";
-
+    private static SimpleDateFormat dateFormat = new SimpleDateFormat(JSON_DATE_FORMAT);
```

Comment:
Due to thread safety issues you really don't want to `new SimpleDateFormat` at the class level.  Creating the format should be done at the method level so multiple threads wont access the same object concurrently.